core/views/irap_request.py

```diff
@@ -175,20 +175,63 @@ def can_view_all(self):
             ]
         )
 
+    def get_url_filters(self) -> dict[str, Any]:
+        params = {}
+        submitted = self.request.GET.get("submitted", None)
+        if submitted:
```

Comment:
Great opportunity for a walrus operator!

```suggestion
        if submitted := self.request.GET.get("submitted", None):
```